src/transformers/models/wav2vec2/modeling_wav2vec2.py

```diff
@@ -1871,6 +1871,11 @@ def forward(
 @add_start_docstrings(
     """Wav2Vec2 Model with a `language modeling` head on top for Connectionist Temporal Classification (CTC).""",
     WAV_2_VEC_2_START_DOCSTRING,
+    """
+        target_lang (`str`, *optional*):
+            Language id of adapter weights. Adapter weights are stored in the format adapter.<lang>.safetensors or
+            adapter.<lang>.bin If None, uses 'eng'.
```

Comment:
may I know why there is no `Only relevant when using an instance ...` part here?

Reply:
not intentional. thanks for the catch!